src/FacebookAds/Object/AbstractObject.php

```diff
@@ -45,6 +45,8 @@ public function __construct() {
   public function __get($name) {
     if (array_key_exists($name, $this->data)) {
       return $this->data[$name];
+    } elseif (in_array($name, static::getFieldsEnum()->getValues())) {
```

Comment:
It does not work.
It seems that `isValid` use the same keys as `getValuesMap` witch exclude field not returned by the api.

I've tried several ways but only found this O(n) solution.
Any other idea?

_ps. don't you have any method to try to reproduce this — maybe unit test, this should be more efficient than suggesting fix ;-)_